src/services/paramValidation.ts

```diff
@@ -18,22 +17,24 @@ export const routeParamsAreValid: RouteMatchRule = (route, url) => {
 }
 
 export const getRouteParamValues = (route: Route, url: string): Record<string, unknown> => {
-  const { pathname, search } = parseUrl(url)
+  const { protocol, host, pathname, search, hash } = parseUrl(url)
 
   return {
+    ...getPathParams(route.host, `${protocol}//${host}`),
```

Comment:
pretty sure this is a bug in production right now, `host` advertises support for params but doesn't validate them when matching

Reply:
Should this be `getHostParams`?

Reply:
if we reuse "path" functions, rename without the word "path".

Reply:
investigate if we actually need a "query" version as well